### PR TITLE
chore: refactor cli docs page

### PR DESCRIPTION
**Pull request by sweatybridge**
## What kind of change does this PR introduce?

docs update

## What is the new behavior?

Reorganizes local dev guide to focus on schema migrations.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 1]
```diff
@@ -1732,7 +1732,7 @@ export const local_development: NavMenuConstant = {
       name: 'Local development',
       url: undefined,
       items: [
-        { name: 'Local development', url: '/guides/local-development/overview' },
+        { name: 'Schema migrations', url: '/guides/local-development/schema-migrations' },
```
Comment (charislam) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:1735]:
```suggestion
        { name: 'Local development', url: '/guides/local-development/overview' },
```

I propose reverting this change, reasoning: we have a good chunk of users who are not familiar with a schema migration workflow, and naming this "schema migrations" pretty much guarantees they won't read it. I think if we call the document "overview", but in the actual document, stress that "schema migrations _are_ the way to do local development", we'll be more likely to educate users.

File: apps/docs/content/guides/local-development/schema-migrations.mdx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
-id: 'local-development'
-title: 'Local Development'
-description: 'How to use Supabase on your local development machine.'
-subtitle: 'How to use Supabase on your local development machine.'
+id: 'schema-migrations'
+title: 'Schema Migrations'
+description: 'How to manage schema migrations for your Supabase project.'
+subtitle: 'How to manage schema migrations for your Supabase project.'
```
Comment (charislam) [inline apps/docs/content/guides/local-development/schema-migrations.mdx:5]:
```suggestion
title: 'Local development with schema migrations'
description: 'Develop locally with the Supabase CLI and schema migrations.'
subtitle: 'Develop locally with the Supabase CLI and schema migrations.'
```

+ revert the filename change

File: apps/docs/content/guides/local-development/schema-migrations.mdx [hunk 3]
```diff
@@ -11,160 +11,6 @@ Supabase is a flexible platform that lets you decide how you want to build your
 
 Doing things directly on the platform via the [Dashboard](https://app.supabase.io/) is fine when you're getting started, but it's a good idea to move to a proper local workflow before you get too far. Working locally, generating migrations as you change your tables, and applying those migrations to a linked project on the [Platform](https://app.supabase.io/) keeps everything nicely organized as you grow.
```
Comment (charislam) [inline apps/docs/content/guides/local-development/schema-migrations.mdx:12]:
```suggestion
Develop locally using the CLI to run a local Supabase stack. You can use the integrated Studio Dashboard to make changes, then capture your changes in schema migration files, which can be saved in version control.

Alternatively, if you're comfortable with migration files and SQL, you can write your own migrations and push them to the local database for testing before sharing your changes.
```

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts (hunk 1 above)
Comment (sweatybridge) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:1735]: Actually the `Local development` title is repeated twice in the nav bar.

image

How about naming it `Getting started` to be consistent with CLI > Getting started or simply `Overview`?

Comment (charislam) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:1735]: Ooops completely forgot that one existed. I like Getting Started!

**Pull request merged**